tests/test_config.py

```diff
@@ -285,3 +285,15 @@ def test_always_use_configured_logger():
     c.set('statsd_host', 'localhost:12345')
     # still uses custom logger over statsd
     assert c.logger_class == MyLogger
+
+def test_load_enviroment_variables_config():
+    os.environ["GUNICORN_CMD_ARGS"] = "--workers=4"
+    with AltArgs():
+        app = NoConfigApp()
+        assert app.cfg.workers == 4
+
+def test_cli_overrides_enviroment_variables_module():
```

Comment:
Tests are look great, thank you! :) Could you also add a test that passes an invalid config option (e.g `os.environ["GUNICORN_CMD_ARGS"] = "--monty-python"`?